styles/_mote.less

```diff
@@ -1,7 +1,7 @@
 .svarside__introTittel {
   .typo-innholdstittel-mixin();
   margin-top: 0;
-  margin-bottom: .5em;
+  margin-bottom: .5rem;
```

Comment:
Hvorfor `rem` i stedet for `em`?

Reply:
Resten av .less fila ser ut til å bruke `rem`, ser ingen visuelle forskjeller mellom `rem` og `em` i dette tilfellet. Kan endre tilbake dersom det foretrekkes `em`

Reply:
Vi pleier å bruke `em`, jeg vet ikke hvorfor det er brukt `rem` noen steder 🤷‍♂